### PR TITLE
[CCOM] Metatag token

### LINKED ISSUE #7522: Add parent org to page title meta tags if set
May 4th  we will be changing the College of Medicine branding over to the block Iowa brand these changes can't be made prior to the 4th.  All current branding changes to the current ccom brand remain other than changing from the healthcare logo to block Iowa and the link of the block Iowa going to uiowa.edu instead of uihealthcare.org
```[tasklist]
### Proposed Solution
- [ ] Possibly create new token for parent org https://github.com/uiowa/uiowa/issues/7522#issuecomment-2061950265
- [ ] Add parent org to page title meta tags if set
- [ ] Should be able to deploy this in advance of May 4th timeline because it won't affect the CCOM brand just the Iowa brand
```

### PR DESCRIPTION
**Pull request by GaryRidgway**
tracks https://github.com/uiowa/uiowa/issues/7522

## To test

1. Sync down something like `neurosurgery.medicine.uiowa.edu`
2. Check that the title of the page (The one in the`<head>` element mirrors the one displayed in your browser tab) now displays both the parent org and the university of iowa.
3. Toggle parent org on and off, check that all combinations of parent org work
4. [Toggle 'Branding options' appearance setting](https://medicineneurosurgery.uiowa.ddev.site/admin/appearance/settings/uids_base) to different values and ensure that it works for all of them with step 3.
5. Try to break this, then comment what you found.

### CODE REVIEW

File: docroot/modules/custom/uiowa_core/uiowa_core.module [hunk 1]
```diff
@@ -1069,25 +1069,36 @@ function uiowa_core_token_info(): array {
...
-
         case 'favicon_assets_path':
           $favicon_path = $ccom_site ? '/profiles/custom/sitenow/assets/uihc_favicons/' : '/profiles/custom/sitenow/assets/';
           $replacements[$original] = $config->get('uiowa_core.favicon_assets_path') ?? $favicon_path;
 
           break;
+
+        case 'combined_org':
+          $site_settings = $config_factory->getEditable('system.site');
+          $formatted_org = 'The University of Iowa';
```
Comment (joewhitsitt) [inline docroot/modules/custom/uiowa_core/uiowa_core.module]: We still need the `$config->get('uiowa_core.org_name')` check for the [regents option](https://iowa.sharepoint.com/sites/drupal-developer-documentation/SitePages/SiteNow%20Drupal%209/Notes%20for%20regent%20option%20websites/Notes%20for%20regent%20option%20websites.aspx).

```suggestion
          $formatted_org = $config->get('uiowa_core.org_name') ?? 'The University of Iowa';
```

Comment (GaryRidgway) [reply docroot/modules/custom/uiowa_core/uiowa_core.module]: Oh true!
Let me check this out

**Comment by joewhitsitt** [comment]
pulling to test

**Review by joewhitsitt — APPROVED** [review]
This looks good.

An existing issue out there to note. The homepage gets a double `The University of Iowa | The University of Iowa`

We may want to follow up with some site-specific handling in that regard.

**Pull request merged**